contrib/tmtplans/buildah_build_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
```

Comment:
```suggestion
set -exo pipefail
```

The order here is important.